rclcpp/src/rclcpp/node_builtin_executor.cpp

```diff
@@ -0,0 +1,188 @@
+// Copyright 2023 Sony Group Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rclcpp/node_builtin_executor.hpp"
+#include "rclcpp/executors/multi_threaded_executor.hpp"
+#include "rclcpp/executors/single_threaded_executor.hpp"
+#include "rclcpp/node_interfaces/node_logging.hpp"
+
+using rclcpp::NodeBuiltinExecutor;
+
+class NodeBuiltinExecutor::NodeBuiltinExecutorImpl
+{
+public:
+  RCLCPP_SMART_PTR_ALIASES_ONLY(NodeBuiltinExecutor)
```

Comment:
It's unnecessary here, please remove it.

Reply:
Yes.